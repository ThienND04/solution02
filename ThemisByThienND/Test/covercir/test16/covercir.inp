4
42443 29312  21941
48644 21864  13705
62117 22981  18467
87189 84157  13143
