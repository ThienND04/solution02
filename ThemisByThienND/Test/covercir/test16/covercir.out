24906
28400
33314
51681
